### DISCUSSION TITLE
Changing Airbyte ports

### DISCUSSION THREAD

**Discussion by aliavni** [airbytehq/airbyte#35451] [Deployment] [answered]
Hi, I would like to map Airbyte's default `800*` ports to something else since it clashes with some other ports in my Docker. 

I tried changing ports in both `.env` and `docker-compose.yaml` files, but I cannot get the UI to load. Is there any other place I need to change default ports?

**Comment by shubham-bhatnagar-78** [comment, marked as answer]
I updated the port under `airbyte-proxy` in` docker-compose.yaml` and it worked, not sure if that's the right way though

**Reply by aliavni** [reply]
It worked. Thanks a lot
